### ISSUE TITLE
pop-up window showing time needs date info as well

### ISSUE THREAD

**Issue by cafinn-usgs** [usgs/geomag-plots#151]
If I am looking at a multi-day time window of data, (for example, Nov 6 through Dec 5) and want to see where a problem occurred, when I hover over the plot with the mouse, it tells me the time, but I have no idea what day it is.  It could be any day, for example, between Nov 13 and Nov 20, the dates shown at the bottom label of the plot.  This is not very helpful.

**Comment by jmfee-usgs** [comment]
In the short term, you can zoom in or out.  We can discuss alternatives including showing the full timestamp, which may obscure other data if it's part of the tooltip.

**Comment by cafinn-usgs** [comment]
I tried that and plots ended up freezing on itself in the process of trying to zoom in and out.  I finally had to give up and reset things to "past 24 hrs".  So until we free up the memory or machine issues that are causing those problems, the zooming is not really a good option.

**Comment by jmfee-usgs** [comment]
That is definitely an issue when looking at a lot of data.  Are you using Google Chrome? it seems to handle more data than other browsers...

**Comment by cafinn-usgs** [comment]
No, I was using firefox and only looking at just under 30 days.
